cmd/cmd.go

```diff
@@ -567,6 +566,7 @@ func Main() {
 		if strings.HasPrefix(err.Error(), "unknown command") && selfupdateEnabled {
 			Root.PrintErrf("You could use '%s selfupdate' to get latest features.\n\n", Root.CommandPath())
 		}
-		log.Fatalf("Fatal error: %v", err)
+		log.Printf("Fatal error: %v", err)
+		os.Exit(exitcode.UsageError)
```

Comment:
@ncw What about this part, is it safe (enough) to assume that whatever ends up here should be exit code 2? I.e. whatever not ending up in resolveExitCode or in any other os.Exit (log.Fatalf) elsewhere. I think typically the `rclone nonexisting` case must be handled here, but not sure what else. The alternative would be to move the change into the "unknown command" if above, and perhaps extend it if needed, but what do you think?